content/applications/sales/members.rst

```diff
@@ -0,0 +1,80 @@
+=======
+Members
+=======
+
+The *Members* application is where all operations related to memberships can be configured and
+managed. The *Members* app integrates with the *Sales* and *Accounting* applications to sell and
+invoice memberships directly to customers.
+
+Membership products
+===================
+
+To create a new membership product, navigate to :menuselection:`Members app --> Configuration -->
+Membership Products` and click :guilabel:`New` to open a blank product record.
+
+Complete the blank form with the necessary information, including the :guilabel:`Membership
+Duration`.
+
+.. note::
+   Membership products require a start and end date, as they are used to determine :ref:`membership
+   status <sales/membership-status>`. Membership products can be sold *before* their active start
+   date.
+
+.. image:: members/membership-product.png
+   :align: center
+   :alt: A new membership product in the members app.
+
+Membership products can be added to a sales order, and invoiced as regular products or
+subscriptions.
+
+Activate a membership
+=====================
+
+To activate a membership from the *Contacts* application, navigate to :menuselection:`Contacts app`
+and click on a contact to open it. Click :guilabel:`Buy Membership` in the :guilabel:`Membership`
+tab of a contact form. On the pop-up window that appears, select a :guilabel:`Membership` from the
+drop-down menu. Then, configure a :guilabel:`Member Price`. Click :guilabel:`Invoice Membership`
+when both fields are filled in.
+
+Alternatively, to offer a free membership, tick the :guilabel:`Free Member` checkbox, in the
+:guilabel:`Membership` tab of a contact form.
+
+.. _sales/membership-status:
+
+Membership status
+=================
+
+The :guilabel:`Current Membership Status` is listed on the :guilabel:`Membership` tab of each
+contact record:
+
+- :guilabel:`Non Member`: a partner who has **not** applied for membership.
+- :guilabel:`Cancelled Member`: a member who has cancelled their membership.
+- :guilabel:`Old Member`: a member whose membership end date has passed.
+- :guilabel:`Waiting Member`: a member who has applied for membership, but whose invoice has not
+  yet been created.
+- :guilabel:`Invoiced Member`: a member whose invoice has been created, but has not been paid.
+- :guilabel:`Paying Member`: a member who has paid the membership fee.
```

Comment:
During my run-through, I complete the invoice process, and when I returned to the contact form (in the membership tab) it read as 'Paid Member' -- not 'Paying Member' --  perhaps these are two separate statuses? Maybe the wording has changed? Either way, I think it would be smart to look into this during your revisions. 
![Screenshot 2024-06-20 130317](https://github.com/odoo/documentation/assets/73958186/e4fffd15-4bbc-4d49-8b90-9e30770971ce)